commit d84ff4edbc48cbab1395d543fda5b1d5ca10570d
Author: Yue Liu <yue.liu@mail.com>
Date:   Sat Nov 17 17:11:12 2012 -0500

    fix WPG detection, patch from Raúl Sánchez Siles <rasasi78@gmail.com>

diff --git a/filters/words/CMakeLists.txt b/filters/words/CMakeLists.txt
index aebcfa1..0c2107c 100644
--- a/filters/words/CMakeLists.txt
+++ b/filters/words/CMakeLists.txt
@@ -57,9 +57,9 @@ IF( NOT TINY )
     macro_log_feature(LIBWPS_FOUND "LibWPS" "Microsoft Works Document Library" "http://libwps.sourceforge.net/" FALSE "" "Required by the Words WPS import filter")
 
     add_subdirectory( applixword )
-    if(WPD_FOUND AND WPG_FOUND)
+    if(WPD_FOUND AND LIBWPG_FOUND)
         add_subdirectory( wordperfect )
-    endif(WPD_FOUND AND WPG_FOUND)
+    endif(WPD_FOUND AND LIBWPG_FOUND)
     if(WPD_FOUND AND LIBWPS_FOUND)
         add_subdirectory( works )
     endif(WPD_FOUND AND LIBWPS_FOUND)
